rest/method_get.go

```diff
@@ -58,6 +59,20 @@ func listGet(ctx context.Context, r *http.Request, route *RouteMatch) (status in
 		e = NewError(err)
 		return e.Code, nil, e
 	}
+	// Now we can set Total.
+	list.Total = len(list.Items)
+
+	// Set appropriate fields.
+	if page >= 0 {
+		list.Page = page + 1
+		list.Skip = ((page + 1) * limit) - limit
```

Comment:
`Skip` should be equal to the `skip` parameter. What are you calculating here? I don't follow.